plugins/src/android/optimove.gradle

```diff
@@ -35,4 +37,18 @@ buildscript {
         
         
     }
+}
+ext.postBuildExtras = {
+    if (file("./google-services.json").exists()) {
+        if (project.extensions.findByName('googleServices') == null) {
+            // apply plugin: 'com.google.gms.google-services'
+            // class must be used instead of id(string) to be able to apply plugin from non-root gradle file
+            apply plugin: com.google.gms.googleservices.GoogleServicesPlugin
+        }
+    }
+    dependencies {
```

Comment:
in kumulos cordova sdk we had

```
def firebaseBomVersion = '28.2.0'
dependencies {
        if (project.hasProperty('KumulosFirebaseBoMVersion')){
            firebaseBomVersion = project.ext.get("KumulosFirebaseBoMVersion")
        }

        implementation platform("com.google.firebase:firebase-bom:$firebaseBomVersion")
        implementation 'com.google.firebase:firebase-messaging'
    }
```

the point of `KumulosFirebaseBoMVersion` was letting consumers of the SDK override bom version to their liking. Do we need smth similar @cgwyllie ?

Reply:
Hm, that's a good question.

Perhaps we should replicate the style we had in Kumulos to allow overriding as you say.

We could theoretically add it later if someone has a conflict, but doesn't seem like a large change to make so perhaps good to do now.

Did we document how to set/override it, and should we add to Optimove docs when we have them?

Reply:
sure, yeah I'll add it to the docs also

Reply:
done :)